Tasks/UseDotNetV2/task.json

```diff
@@ -44,13 +44,31 @@
                 "sdk": "SDK (contains runtime)"
             }
         },
+        {
+            "name": "useGlobalJson",
+            "type": "boolean",
+            "label": "Use global json",
+            "defaultValue": false,
+            "required": false,
+            "helpMarkDown": "Select this option to install all SDKs from global.json files. These files are searched from system.DefaultWorkingDirectory. You can change the search root path by setting working directory input.",
+            "visibleRule": "packageType = sdk"
+        },
+        {
+            "name": "workingDirectory",
+            "type": "filePath",
+            "label": "Working Directory",
+            "helpMarkDown": "Select this option to install all SDKs from global.json files. These files are searched from system.DefaultWorkingDirectory. You can change the search root path by setting working directory input.",
```

Comment:
Specify path from where global.json files should be searched when using `Use global json`. If empty, `system.DefaultWorkingDirectory` will be considered as the root path.